src/integration.jl

```diff
@@ -168,3 +168,47 @@ function tight_remainder(f, tm::T, xm::T, x0::Interval, max_steps::Integer=20) w
     end
     return xOld
 end
+
+
+
+function integrate(f::TaylorModelN, which=1, x0=0)
+
+    p = integrate(f.pol, which)  # not necessary if an already complete Taylor series, in which case p2 == f.p
+
+    Δ = integral_bound(f, which)
+
+    g = TaylorModelN(p, Δ, f.x0, f.I)
+
+    g.pol[0] += x0  # constant term
+
+    # for k in 0:(x0.order)
+    #     TaylorSeries.add!(g.p, g.p, x0, k)
+    # end
+
+    return g
+
+end
+
+
+
+"""
+Bound the integral of a `TaylorNModel` `f` with respect to the variable `which`.
+"""
+function integral_bound(f::TaylorModelN, which)
+
+    high_order_term = f.pol[end]  # a HomogeneousPolynomial
+
+    B = bound(high_order_term, f.x0, f.I)
+    Δ = f.rem
+    w = diam(f.I[which])
+
+    @show B, Δ, w
+
+    Δ_new = ( B + Δ ) * w
+
+    return Δ_new
+end
+
+
+bound(f::HomogeneousPolynomial, x0, I) = f( [(I - x0)...] )
```

Comment:
I think `bound` is equivalent to `evaluate`. So you can write (above) `B = high_order_term(f.I-f.x0)`.